api/templates/teledeclaration_pdf.html

```diff
@@ -83,6 +83,15 @@ <h3>
             Vous avez choisi de déclarer les données d'approvisionnement au nom de toutes vos cantines satellites.
         </p>
         {% endif %}
+        {% if teledeclaration_mode == "CENTRAL_ALL" or teledeclaration_mode == "CENTRAL_APPRO" %}
```

Comment:
À noter qu'on ne peut pas utiliser un if avec des parenthèses comme `{% if a or (b and c) %}`, donc le double if me semble plus clair.